### PR TITLE
Generate YARD docs locally

### PR DESCRIPTION
**Pull request by ms-ati**
Run 'bundle exec yard server' to run a local server to
iterate on documentation and see the results in a browser.

### CODE REVIEW

**Comment by ms-ati** [comment]
Relates to #34

File: README.md [hunk 1]
```diff
@@ -60,14 +60,30 @@ release a new version, update the version number in `version.rb`, and then run
...
+## Documentation
+
+We maintain API documentation using [YARD](https://yardoc.org/), which is
+published automatically at
+[RubyDoc.info](https://rubydoc.info/github/panorama-ed/memo_wise/main).
+
+To edit documentation locally and see how it will be rendered, try running:
+
+```bash
+bundle config set with 'docs'
```
Comment (ms-ati) [inline README.md:72]:
```suggestion
bundle config set with 'docs:linters'
```

File: README.md [hunk 2]
```diff
@@ -60,14 +60,30 @@ release a new version, update the version number in `version.rb`, and then run
 git commits and tags, and push the `.gem` file to
 [rubygems.org](https://rubygems.org).
 
+## Documentation
+
+We maintain API documentation using [YARD](https://yardoc.org/), which is
+published automatically at
+[RubyDoc.info](https://rubydoc.info/github/panorama-ed/memo_wise/main).
+
+To edit documentation locally and see how it will be rendered, try running:
```
Comment (JacobEvelyn) [inline README.md:69]: Nitpick: `try running` to me sounds like we're not confident these steps will work. Can we remove `try`?

**Review by JacobEvelyn — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ms-ati) [reply README.md:69]: Yes! Thanks

**Review by JacobEvelyn — APPROVED** [review]

**Pull request merged**